examples/a_treeple/treeple_tutorial_pvalue.py

```diff
@@ -0,0 +1,186 @@
+"""
+========================================
+Treeple tutorial for calculating p-value
+========================================
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+from scipy.stats import entropy
+
+from sktree.datasets import make_trunk_classification
+from sktree.ensemble import HonestForestClassifier
+from sktree.stats import build_hyppo_oob_forest
+
+# %%
+# Independence Testing
+# --------------------
+#
+# Given samples from ``X`` and ``Y``, the independent hypothesis and its
+# alternative are stated as:
+#
+# .. math:: H_0 : F_{XY} = F_X F_Y
+#
+# .. math:: H_A : F_{XY} \neq F_X F_Y
+#
+# By computing the p-value using ``treeple``, we can test if :math:`H_0`
+# would be rejected, which confirms the joint distribution. The p-vlaue is
```

Comment:
which would confirm that X and Y are not independent. Also, p-value is spelled wrong